### DISCUSSION TITLE
能否使用非80和443端口？

### DISCUSSION THREAD

**Discussion by szh811218** [alibaba/higress#907] [General]
我在测试的时候需要使用80和443端口的同时，还需要使用9999端口访问，但是域名配置只能配置80和443，higress支持增加其他端口嘛？有没有大佬可以教我？

**Comment by CH3CHO** [comment]
机制上肯定是可以的，不过这就要你自己改一些配置了，比如K8s Service里暴露的端口，路由的配置也没法使用常规的Ingress了，可以考虑用Gateway API或者Istio CRD。

**Reply by szh811218** [reply]
明白了，谢谢
